.github/workflows/actions-audit.yaml

```diff
@@ -1,18 +0,0 @@
-# This workflow is managed by application-workflows; any changes will be lost.
```

Comment:
This will be restored with https://github.com/RentTheRunway/application-workflows/pull/284